### DISCUSSION TITLE
Is it safe to use Alert?

### DISCUSSION THREAD

**Discussion by 15MariamS** [microsoft/fluentui#29228] [General] [answered]
Wondering if the API is stable enough to use [Alert](https://react.fluentui.dev/?path=/docs/preview-components-alert--default)?

**Comment by layershifter** [comment, marked as answer]
Not really 🐱 `Alert` will be deprecated & replaced with a `MessageBar` component. ETA for `MessageBar` is end of October.

**Comment by pitric** [comment]
Was just going to ask the same question. Will Alert still work until the end of October and any time after?

**Reply by layershifter** [reply]
The package is published, so you should be able to use it, but Alert will be deprecated, so no fixes/releases are expected.
